src/gwt/src/org/rstudio/studio/client/common/spelling/domain_specific_words.csv

```diff
@@ -0,0 +1,456 @@
+coloured
+colouring
+colourised
+colours
+config
+cpp
+cran
+csl
+css
+csv
+ctrl
+cy
+da
+datepicker
+datetime
+de
+delim
+depsy
+dev
+devtools
+dir
+dircolors
+disqus
+distill's
+dockerhub
+docsearch
+doi
+dowle
+dplyr
+dropbox
+drs
+dvipsnames
+Eggenberger
+el
+emacs
+env
+envvar
+eo
+errors
+et
+etag
+etags
+eu
+eval
+extdata
+facebook
+favicon
+fcntl
+fi
+fieldguide
+filenaming
+filesystem
+finalizer
+flexdashboard
+fo
+fontenc
+fontspec
+foofy
+forkable
+françois
+fread
+frontend
+frontmatter
+fs
+funder
+fwf
+gabe
+gc
+gcc
+getgrent
+getpwent
+ggplot
+gists
+git's
+github
+gitignore
+gitignores
+gitlab
+gl
+globbing
+Gompertz
+grdevices
+grisu
+gui
+gz
+gzip
+gzipped
+hadley
+hardcoding
+hexb
+hexidecimal
+homebrew
+hostname
+http
+https
+httr
+hu
+hy
+hypergeometric
+ides
+ietf
+iframe
+igraph
+initialisation
+initialise
+initialises
+initialising
+inode
+installable
+instapaper
+interruptible
+io
+ish
+issn
+isvtx
+ja
+javascript
+jenkinsfile
+joyent
+json
+jupyter
+ka
+kalibera
+keychain
+kh
+kib
+kk
+knitr
+ko
+kr
+labelled
+lbzip
+learnr
+lexicographically
+lgl
+libbsd
+libgit
+libuv
+lifecycle
+linkedin
+linkingto
+linters
+lintr
+linux
+localtime
+lockfile
+logit
+longtable
+lpt
+lt
+lualatex
+lv
+ly
+macos
+magrittr
+magrittr's
+mainfont
+maintainer's
+makefile
+makefiles
+mandreyel
+mathfont
+md
+metadata
+microtype
+minimise
+mio
+misspecify
+mk
+mn
+modelled
+monofont
+morem
+msg
+mtcars
+multithreaded
+nano
+natbib
+natively
+navbar
+navbars
+nb
+nd
+nfs
+nl
+nodejs
+noncentral
+notes
+nulls
+numpy
+nyc
+nycflights
+objs
+oneside
+only’
+openssl
+orcid
+organisation
+organisations
+ostype
+pagedown
+pageins
+pak's
+pak’s
+pandoc
+parsers
+pats
+pdf
+pdflatex
+pid
+pids
+pigz
+pinterest
+pixz
+pkgbuild
+pkgdown
+png
+pollable
+Polya
+posixt
+pre
+preconfigured
+preprocess
+processx
+programmatically
+prs
+pthread
+pubkey
+purrr
+px
+r's
+Rademacher
+rchk
+rcpp
+rcpparmadillo
+rcppeigen
+rdatatable
+rds
+readline
+readme
+readr
+realisation
+rebase
+recognised
+recurse
+redirections
+renviron
+repl
+repo
+repo's
+report
+repos
+reprex
+reproducibility
+restructing
+retryable
+revdep
+revdepcheck
+revdepchecks
+revealjs
+rhub
+rlang
+rmarkdown
+rmd
+rmds
+ro
+roadmap
+roclet
+romain
+roxygen
+rprofile
+rprojroot
+rray's
+rstd
+rstudio
+rstudio's
+rstudio’s
+rticles
+rtools
+ru
+runnable
+sansfont
+searchability
+setspace
+sha
+shortener
+shorteners
+shorthands
+shortlink
+shortlinks
+shortname
+shr
+sigcont
+sigkill
+signalling
+sitrep
+situation
+sk
+Skellam
+sl
+soliton
+sr
+src
+srcrefs
+stdout
+strftime
+stringi
+styler
+stylesheet
+subcommand
+subprocess
+subprocesses
+subsetters
+subsubsection
+subtag
+sunsetted
+sv
+svgnames
+sw
+symlink
+symlinks
+synchronised
+sys
+templated
+templating
+testthat
+tex
+th
+tibble
+tibbles
+tidyverse
+tidy’
+tierney
+toc
+todo
+tokenizer
+toml
+toolchain
+tooltip
+traceback
+travis
+triaged
+trs
+tsv
+Tukey
+tw
+tz
+ubsan
+udp
+ui
+uid
+uk
+un
+unk
+unparsed
+user
+usethis
+usethis's
```

Comment:
I'm surprised these are considered distinct words by the spell checker. Should we just allow plural forms of any word in the domain list?

Reply:
Most of these are pulled directly from the WORDS list from popular R packages list and not hand curated besides the few that I manually added. There are few enough that I can hand trim them.